dev/README.mdx

```diff
@@ -0,0 +1,23 @@
+### Using `get_changes`
```

Comment:
why `README.mdx`? Is that a typo or should it be like that?

Reply:
Not a typo, but MDX is probably overkill. MD will work, and I will change it back.